src/slice_transform.rs

```diff
@@ -118,6 +111,46 @@ pub unsafe extern "C" fn in_domain_callback(
 ) -> u8 {
     let cb = &mut *(raw_cb as *mut TransformCallback);
     let key = slice::from_raw_parts(raw_key as *const u8, key_len as usize);
-    let in_domain = cb.in_domain_fn.unwrap();
-    in_domain(key) as u8
+    if let Some(in_domain) = cb.in_domain_fn {
+        in_domain(key) as u8
+    } else {
+        0xff_u8
```

Comment:
Nit: The postfix `_u8` is needless because the compiler knows a type of a return value of the function.